numerics_paper/version2/numerical_chj.tex

```diff
@@ -257,7 +256,7 @@ \subsection{Wave propagation methods}\label{sec:waveprop}
 
 
 \subsection{Rusanov's Riemann solver}\label{sec:rusanov}
-The Rusanov solver approximates the Riemann solution by with a single wave traveling in each
+The Rusanov's solver approximates the Riemann solution by with a single wave traveling in each
```

Comment:
It's correct to say either "Rusanov's solver" or "The Rusanov solver", but not "The Rusanov's solver".  The most common term in my experience is "the Rusanov solver".

Reply:
Corrected.